src/core/runtime/src/configuration/load-organizations-config.ts

```diff
@@ -388,13 +388,25 @@ async function validateScpsCount(
       Filter: 'SERVICE_CONTROL_POLICY',
       TargetId: ouObject.Id!,
     });
-    const accelScps = ouConfig.scps.map(policyName =>
+    const accelOuScps = ouConfig.scps.map(policyName =>
       ServiceControlPolicy.policyNameToAcceleratorPolicyName({ acceleratorPrefix, policyName }),
     );
+    const configScps = config['global-options'].scps;
+    const accelScps = configScps.map(scp =>
```

Comment:
FYI: This fix only works if the customer leaves the old SCPs defined in global options.

Reply:
Agreed, however it enable them to more easily switch OU's if necessary in the future.